stock_lot_scrap/models/stock_scrap_lot.py

```diff
@@ -4,14 +4,22 @@
 
 
 from openerp import api, models
-from openerp.exceptions import Warning
+from openerp.exceptions import Warning as UserError
 from openerp.tools.translate import _
 
 
 class StockProductionLot(models.Model):
     _inherit = 'stock.production.lot'
 
     @api.multi
+    def _prepare_picking_vals(self, warehouse):
+        return {
+            'origin': _('Lot: %s') % self.name,
+            'picking_type_id': warehouse.int_type_id.id or self.env.ref(
+                'stock.picking_type_internal').id,
+        }
+
+    @api.multi
     def _prepare_move_vals(self, picking, quant, strap_location_id):
```

Comment:
s/strap_location_id/scrap_location_id (replace it in the 4 places that it appears)